console_conf/controllers/identity.py

```diff
@@ -136,8 +126,8 @@ def start_ui(self):
         device_owner = get_device_owner()
         if device_owner is not None:
             self.model.add_user(device_owner)
-            key_file = os.path.join(device_owner['homedir'],
```

Comment:
at the moment device_owner on unmanaged device, with seeded lxd snap will be `lxd`. It should be empty. Hence get_device_owner() must check for `snap managed`.